.github/actions/setup-indy-pool/action.yml

```diff
@@ -6,6 +6,9 @@ inputs:
   seed:
     description: Seed to register on the ledger
     required: true
+  endorserSeed:
```

Comment:
we already have a seed registered with endorser role right? Why do we need an extra one?

Reply:
"unknownError: Error: Request failed: client request invalid: UnauthorizedClientRequest(\"Endorser must have one of the following roles: {'ENDORSER'}\",)"

I get this error when I try to endorse transactions with the trustee role. Maybe I am doing something wrong?

Reply:
Ah okay. And if we switch trustee to endorser for the main seed?